modules/market/src/main/java/com/opengamma/strata/market/value/IborIndexRates.java

```diff
@@ -90,6 +89,20 @@ public default IborIndexRatesKey getKey() {
    * @throws RuntimeException if the value cannot be obtained
    */
   public abstract double rate(LocalDate fixingDate);
+  
+  /**
+   * Gets the forward rate at the specified fixing date.
+   * <p>
+   * This method bypasses the potential historical time series associated to the index. 
+   * The standard method to access the Ibor rate is {@link IborIndexRates#rate}; this method should be used only 
+   * for specific purposes, when the actual time series are not relevant. 
+   * <p>
+   * In case there is a doubt on using this method, it probably means that it should not be used.
+   * 
+   * @param fixingDate  the fixing date to query the rate for
+   * @return the rate of the index as given by the forward curve
+   */
+  public abstract double forwardRate(LocalDate fixingDate);
```

Comment:
This name might be too appealing for callers. Perhaps `rateForwardOnly()` ? Or `rateIgnoringTimeSeries` ?